services/consensus_service.proto

```diff
@@ -31,93 +38,89 @@ import "transaction_response.proto";
+     * Create an HCS topic.
+     * <p>
+     * On success, the resulting TransactionReceipt SHALL contain the newly
+     * created TopicId.<br/>
+     * If the `adminKey` is set on the topic, this transaction MUST be signed
+     * by that key.<br/>
+     * If the `adminKey` is _not_ set on the topic, this transaction MUST NOT
+     * set an `autoRenewAccount`. The new topic will be immutable and must be
+     * renewed manually.<br/>
+     * If the `autoRenewAccount` is set on the topic, this transaction MUST be
+     * signed by that account.<br/>
+     * <p>
+     * The request body MUST be a
+     * [ConsensusCreateTopicTransactionBody](#proto.ConsensusCreateTopicTransactionBody)
      */
     rpc createTopic (Transaction) returns (TransactionResponse);
 
     /**
-     * Update a topic.
-     * If there is no adminKey, the only authorized update (available to anyone) is to extend the expirationTime.
-     * Otherwise transaction must be signed by the adminKey.
-     * If an adminKey is updated, the transaction must be signed by the pre-update adminKey and post-update adminKey.
-     * If a new autoRenewAccount is specified (not just being removed), that account must also sign the transaction.
-     * Request is [ConsensusUpdateTopicTransactionBody](#proto.ConsensusUpdateTopicTransactionBody)
+     * Update an HCS topic.
+     * <p>
+     * If the `adminKey` is not set on the topic, this transaction MUST extend
+     * the `expirationTime` and MUST NOT modify any other field.<br/>
+     * If the `adminKey` is set on the topic, this transaction MUST be signed
+     * by that key.<br/>
+     * If this transaction sets a new `adminKey`, this transaction MUST be
+     * signed by <strong>_both_</strong> keys, the pre-update `adminKey` and
+     * the post-update `adminKey`.<br/>
+     * If this transaction sets a new, non-null, `autoRenewAccount`, the newly
+     * set account MUST sign this transaction.<br/>
+     * <p>
+     * The request body MUST be a
+     * [ConsensusUpdateTopicTransactionBody](#proto.ConsensusUpdateTopicTransactionBody)
      */
     rpc updateTopic (Transaction) returns (TransactionResponse);
 
     /**
-     * Delete a topic. No more transactions or queries on the topic (via HAPI) will succeed.
-     * If an adminKey is set, this transaction must be signed by that key.
-     * If there is no adminKey, this transaction will fail UNAUTHORIZED.
-     * Request is [ConsensusDeleteTopicTransactionBody](#proto.ConsensusDeleteTopicTransactionBody)
+     * Delete an HCS topic.
+     * <p>
+     * If this transaction succeeds, all subsequent transactions referencing
+     * the deleted topic SHALL fail.<br/>
+     * The `adminKey` MUST be set on the topic and this transaction MUST be
+     * signed by that key.<br/>
+     * If the `adminKey` is not set on the topic, this transaction SHALL fail
+     * with a response code of `UNAUTHORIZED`. A topic without an `adminKey`
+     * cannot be deleted, but MAY expire.<br/>
```

Comment:
There is no expired Topic. You can update expirationTime without an 'adminKey'.

Reply:
This line is specific to `deleteTopic`.  A delete transaction cannot update expiration time.
The `updateTopic` transaction does note that the expiration time can be updated without the admin key.

Expired Topics are specified, we just haven't enabled that functionality yet.

Reply:
Again if these comments need to match the current implementation?